app/controllers/api/v1/organizations_controller.rb

```diff
@@ -0,0 +1,52 @@
+# :reek:InstanceVariableAssumption
+class Api::V1::OrganizationsController < ApplicationController
+  before_action :set_organization, only: %i[show update]
+  before_action :authenticate_user!
+  before_action :unauthorized_user, only: %i[create update]
+
+  def show
+    if @organization && current_user.organizer? && organization_owner?
```

Comment:
`organization_owner?` also can be moved to user model

Reply:
but organization_owner? checks if current_user.organization identical to the current organization, how can i move it to model?

Reply:
I'd say better to move to decorator